community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -237,13 +246,15 @@ object projects:
 
   lazy val scas = MillCommunityProject(
     project = "scas",
-    baseCommand = "scas.application"
+    baseCommand = "scas.application",
+    requiresExperimental = true
```

Comment:
`scas` does not seem to depend on experimental.

Reply:
Yep, looking at the commit history that's correct; this was an oversight. Other projects required experimental previously as far as I can tell.